README.md

```diff
@@ -103,6 +103,7 @@ If you're having issues with the packaged executable or using this tool in gener
 
 | Language                                    | Code  | Translator                                    | Translated                           |
 | ------------------------------------------- | ----- | --------------------------------------------- | ------------------------------------ |
+| Arabic (العربية)                              | ar   | [Tayeb-Ali](https://github.com/tayeb-ali)      | UI                                   | 
```

Comment:
Looks like this might need to be shifted over a bit more

Reply:
@Tayeb-Ali it looks like the changes you made didn't align this one up

Reply:
> @Tayeb-Ali it looks like the changes you made didn't align this one up
It was fixed in the last update below this comment.
Commit ID = [d73ed60](https://github.com/brentvollebregt/auto-py-to-exe/pull/319/commits/d73ed6011c6b776c538077699ac2aae5c21316f9)

Reply:
These were the changes in d73ed6011c6b776c538077699ac2aae5c21316f9

```diff
  | Language                                    | Code  | Translator                                    | Translated                           |
  | ------------------------------------------- | ----- | --------------------------------------------- | ------------------------------------ |
- | Arabic (العربية)                              | ar   | [Tayeb-Ali](https://github.com/tayeb-ali)      | UI                                   | 
+ | Arabic (العربية) 	| ar 	| [ Tayeb-Ali ](https://github.com/tayeb-ali) 	| UI |
  | Brazilian Portuguese (Português Brasileiro) | pt_br | [marleyas](https://github.com/marleyas)       | UI (partial)                         
  |
  | Chinese Simplified (简体中文)               | zh    | [jiangzhe11](https://github.com/jiangzhe11)   | UI and [README](./README-Chinese.md) |
  | Chinese Traditional (繁體中文)              | zh_tw | [startgo](https://github.com/ystartgo)        | UI                                   |
```

See how the `|`'s are lining up like the others?